eclipse-scout-core/src/modeselector/ModeSelector.js

```diff
@@ -108,6 +125,27 @@ export default class ModeSelector extends Widget {
         visibleModes[selectedModeIndex + 1].$container.addClass('after-selected');
       }
     }
+    this._updateSlider();
+  }
+
+  _updateSlider() {
```

Comment:
It would be nice if the slider could be moved by dragging. Especially on touch devices the user could want to enable it this way. It think it is not necessary to visualize the dragging itself, just move the slider when the finger / mouse is on top of the new mode.